roles/lib_utils/src/class/repoquery.py

```diff
@@ -125,6 +145,9 @@ def repoquery(self):
         else:
             rval['package_found'] = False
 
+        if self.ignore_excluders:
```

Comment:
There are a few things you could do to clean this up. Instead of closing the file if the ```ignore_excluders``` is set, you could do the following from line 112:

```python
if self.ignore_excluders:
    with tempfile.NamedTemporaryFile() as tmp_file:
        with open("etc/yum.conf", "r") as file_handler:
[...]
```

You would no longer need to check and explicitly close the temporary file.